packages/a11y/README.md

```diff
@@ -16,20 +16,16 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 
 <!-- START TOKEN(Autogenerated API docs) -->
 
-### setup
-
-[src/index.js#L16-L26](src/index.js#L16-L26)
+<a name="setup" href="#setup">#</a> **setup** [\<>](src/index.js#L16-L26)
```

Comment:
Were you still planning to apply the code formatting, as per #14290 ?

Reply:
Tried that as well but liked this more.